data/blog/vasil-dimov-receives-lts-grant.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: "Long-Term Support For Vasil Dimov"
+date: '2023-10-16'
+tags: ['OpenSats', 'grants', 'bitcoin', 'LTS']
+draft: false
+authors: ['dergigi']
+images: ['/static/images/blog/10-vasil.jpg']
+summary: "OpenSats Welcomes Vasil Dimov as an LTS Grantee."
+---
+
+We are pleased to add Vasil Dimov to the list of OpenSats LTS grantees. Vasil is
+receiving long-term support for his ongoing contributions to Bitcoin Core.
+
+Vasil started contributing full-time to Bitcoin Core in 2020. Since then, he has
+worked on various improvements, focusing mostly on the networking layer of the
+software. In the last couple of years, he has worked on adding [Tor v3
+support](https://github.com/bitcoin/bitcoin/issues/18884), as well as adding
+support for [I2P](https://github.com/bitcoin/bitcoin/pull/20685) and
+[CJDNS](https://github.com/bitcoin/bitcoin/pull/23077). His [merged
+PRs](https://github.com/bitcoin/bitcoin/pulls?q=is%3Apr+author%3Avasild+is%3Aclosed+sort%3Aupdated-desc)
```

Comment:
Maybe also mention review (it is at least as important as own PRs) and include not yet merged PRs in own PRs list:

```suggestion
[CJDNS](https://github.com/bitcoin/bitcoin/pull/23077). His [review](https://github.com/bitcoin/bitcoin/pulls?q=commenter%3Avasild+-author%3Avasild) and [PRs](https://github.com/bitcoin/bitcoin/pulls?q=is%3Apr+author%3Avasild)
```